packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -4036,13 +4036,11 @@ abstract class Element extends DiagnosticableTree implements BuildContext {
     assert(slots == null || newWidgets.length == slots.length);
 
     Element? replaceWithNullIfForgotten(Element child) {
-      return forgottenChildren != null && forgottenChildren.contains(child) ? null : child;
```

Comment:
forgottenChildren is the most metal variable name I've seen in Flutter.

Reply:
### 😂😂